cmd/topology/topology.go

```diff
@@ -44,8 +60,62 @@ var (
 		Short: "push or generate certs for nodes in topology",
 		RunE:  certFn,
 	}
+	resetCfgCmd = &cobra.Command{
+		Use:   "reset -skip <topology> <device>",
+		Short: "reset configuration of device (if device not provide reset all nodes)",
+		RunE:  resetCfgFn,
+	}
+)
+
+var (
+	skipReset bool
 )
 
+func resetCfgFn(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 || len(args) > 2 {
+		return fmt.Errorf("%s: invalid args", cmd.Use)
+	}
+	topopb, err := topo.Load(args[0])
+	if err != nil {
+		return fmt.Errorf("%s: %w", cmd.Use, err)
+	}
+	s, err := cmd.Flags().GetString("kubecfg")
+	if err != nil {
+		return err
+	}
+	t, err := topo.New(s, topopb)
+	if err != nil {
+		return fmt.Errorf("%s: %w", cmd.Use, err)
+	}
+	ctx := cmd.Context()
+	t.Load(ctx)
+	nodes := t.Nodes()
+	if len(args) > 1 {
+		n, err := t.Node(args[1])
+		if err != nil {
+			return err
+		}
+		nodes = append(nodes, n)
+	}
```

Comment:
This will add the specified node to the entire collection of nodes a second time.

Reply:
grr this why i didn't want to have that pattern